app/src/main/java/com/kamron/pogoiv/updater/AppUpdateLoader.java

```diff
@@ -27,10 +26,19 @@ public void run() {
         try {
```

Comment:
Is it possible to release both apk files under a single release ? I'm asking because I haven't used GitHub releases yet.